extras/scene-tree.reel/scene-tree.js

```diff
@@ -5,7 +5,7 @@
 var Component = require("montage/ui/component").Component,
     Dict = require("collections/dict"),
     SceneTreeFactory = require("./core/scene-tree-factory").SceneTreeFactory,
-    Application = require("montage/core/application").Application,
+    Application = require("montage/core/application").application,
```

Comment:
It doesn't sounds right that the symbol exported fro application is lowercase application